docs/SETUP.md

```
@@ -1,30 +1,27 @@
-+ Ensure you have any Python version in the range **[3.11, 3.13]** installed.
-  + If you don't, download one from [here][python-downloads].
-+ Ensure you have the **uv package manager** installed.  
-  + If you don’t, install it by following the steps [here][uv-install].
-+ Clone or download this repository.
+### 📁 Step 1: Get the source code
++ Get the repository via HTTPS or SSH.
 + Change directory into the project root.
 
-### 📦 Step 2: Install dependencies
+### 🔧 Step 2: Install uv
+Install the **uv package manager** by following the steps [here][uv-install].
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)